lib/index.js

```diff
@@ -48,41 +77,133 @@ function compile_template(template_name, type) {
  * the email has been sent.
  */
 function sendemail(template_name, person, callback) {
-  // see: http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/SES.html#sendEmail-property
-  var params = {
-    Destination: { /* required */
-      ToAddresses: [
-        person.email
-      ]
-    },
-    Message: { /* required */
-      Body: { /* required */
-        Html: {
-          Data: compile_template(template_name, 'html')(person), /* required */
-          Charset: 'utf8'
+    // see: http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/SES.html#sendEmail-property
```

Comment:
I would _much_ sooner re-factor the `sendemail` function to allow `CC` and `BCC` and have a `sendmany` which in turn _uses_ the `sendemail` function.